src/PointwiseFunctions/ScalarTensor/ScalarCharge.hpp

```diff
@@ -0,0 +1,90 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <utility>
+
+#include "ApparentHorizons/StrahlkorperGr.hpp"
+#include "ApparentHorizons/TagsDeclarations.hpp"  // IWYU pragma: keep
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/DataBox/TagName.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/EagerMath/Magnitude.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Evolution/Systems/CurvedScalarWave/Tags.hpp"
+#include "NumericalAlgorithms/SphericalHarmonics/Strahlkorper.hpp"
+#include "NumericalAlgorithms/SphericalHarmonics/TagsDeclarations.hpp"
+#include "NumericalAlgorithms/SphericalHarmonics/TagsTypeAliases.hpp"
+#include "PointwiseFunctions/GeneralRelativity/TagsDeclarations.hpp"  // IWYU pragma: keep
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace ScalarTensor {
+
+/// @{
+/*!
+ * \brief The scalar charge per unit solid angle.
+ *
+ * \details This function calculates the integrand of:
+ * \f{align*}
+ * q = - \dfrac{1}{4 \pi} \oint dA \Phi_i n^{i},
+ * \f}
+ * where \f$ n^{i} \f$ is the unit (outward) normal of the surface.
+ *
+ * For a spherically symmetric scalar, this value will coincide with the value
+ * as extracted from the $r^{-1}$ decay in
+ * \f[ \Psi \sim \phi_\infty + q / r + \cdots~. \f]
+ */
+template <typename Frame>
+void scalar_charge_integrand(
+    const gsl::not_null<Scalar<DataVector>*> result,
+    const tnsr::i<DataVector, 3, Frame>& phi,
+    const tnsr::I<DataVector, 3, Frame>& unit_normal_vector,
+    const Scalar<DataVector>& area_element);
```

Comment:
you can delete the `area_element` argument

Reply:
Removed